shared/rakefiles/xk.rake

```diff
@@ -65,6 +65,10 @@ task :xk, [:cmd, :skip_secret_mgmt] => [:configure_serviceaccount, @kubectl_cred
 
   Secrets.set_secrets(@secrets)
 
+  # TODO: Next line should be removed once Terraform issue with GCS backend encryption is fixed
+  # https://issues.gpii.net/browse/GPII-3329
+  ENV['GOOGLE_ENCRYPTION_KEY'] = ENV['TF_VAR_key_tfstate_encryption_key']
```

Comment:
Moving this to `set_secrets()` might be a better guarantee that everything that uses secrets gets all the right configuration.

Reply:
Good point, thanks!